parlai/agents/bert_ranker/helpers.py

```diff
@@ -116,7 +116,7 @@ def forward(self, token_ids, segment_ids, attention_mask):
         elif self.aggregation == "max":
             #  consider the max of all the output except CLS
             outputs_of_interest = embedding_layer[:, 1:, :]
-            mask = (attention_mask[:, 1:].float().unsqueeze(2) - 1) * 10000
+            mask = (attention_mask[:, 1:].float().unsqueeze(2) - 1) * neginf(dtype)
```

Comment:
I do not confirm. That line might use a comment that is true: here the attention mask is 1 if present and 0 if not. Your modification make it +infinity when the token is not present

Reply:
ahh so `attention_mask - 1` is either 0 or -1, which is then multiplied by 10000, to produce a value that's either -10000 or 0.

So this could be fixed with `~attention_mask * neginf`, right?

Reply:
Is this better now @samhumeau ?